sqlmesh/core/snapshot/evaluator.py

```diff
@@ -321,6 +321,21 @@ def audit(
 
         logger.info("Auditing snapshot %s", snapshot.snapshot_id)
 
+        if wap_id is not None:
+            deployability_index = deployability_index or DeployabilityIndex.all_deployable()
+            original_table_name = snapshot.table_name(
+                is_deployable=deployability_index.is_deployable(snapshot)
+            )
+            wap_table_name = self.adapter.wap_table_name(original_table_name, wap_id)
+            logger.info(
+                "Auditing WAP table '%s', snapshot %s", wap_table_name, snapshot.snapshot_id
+            )
+
+            table_mapping = kwargs.get("table_mapping") or {}
+            table_mapping[snapshot.name] = wap_table_name
```

Comment:
@eakmanrq please note that this will have to be reflected in your PR if this gets merged eariler.